exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A"; 
+  }else if(mark < 80 && mark > 60){
```

Comment:
Here, you don't need the first condition: `mark < 80`. It will only pass into this else if condition if it is already smaller than 80 (becuse it has to fail the first if statement).